certora/specs/HashConsensus.spec

```diff
@@ -0,0 +1,742 @@
+/**************************************************
+ *               Methods Declaration              *
+ **************************************************/
+methods {
+    submitConsensusReport(bytes32 report, uint256 refSlot, uint256 deadline) => CONSTANT
+    getLastProcessingRefSlot() returns (uint256) => CONSTANT
+    getConsensusVersion() returns (uint256) => CONSTANT
+
+    reportReceivedCounter(uint256 refSlot, address member, bytes32 report) => getSuccessfulReportSubmitEventInfo(refSlot, member, report)
+}
+
+/**************************************************
+ *                GHOSTS AND HOOKS                *
+ **************************************************/
+// the three ghost variables below store info about a successful report submit
+ghost uint256 ghostReportRefSlot {
+    init_state axiom ghostReportRefSlot == 0;
+}
+ghost address ghostReportMember {
+    init_state axiom ghostReportMember == 0;
+}
+ghost bytes32 ghostReportHash {
+    init_state axiom ghostReportHash == 0;
+}
+
+
+/**************************************************
+ *                CVL FUNCS & DEFS                *
+ **************************************************/
+function saneTimeConfig() returns uint256 {         // returns the timestamp when the valid setting are set
+    env e0; calldataarg args0;
+
+    uint256 correctInitialEpoch;
+    updateInitialEpoch(e0, correctInitialEpoch);    // must be called after constructor
+
+    uint256 slotsPerEpoch; uint256 secondsPerSlot; uint256 genesisTime;
+    slotsPerEpoch, secondsPerSlot, genesisTime = getChainConfig(e0);
+    require slotsPerEpoch == 32;                    // simplification, must be required at constructor
+    require secondsPerSlot == 12;                   // simplification, must be required at constructor
+    require genesisTime < e0.block.timestamp;       // safe assumption, must be required at constructor
```

Comment:
If e0 is a local env variable of this function, what is the meaning of this requirement, in a scope of any rule?
I think you can omit it.